src/cart/crt_context.c

```diff
@@ -1071,7 +1038,6 @@ crt_context_req_track(struct crt_rpc_priv *rpc_priv)
 	/* add the RPC req to crt_ep_inflight */
 	D_MUTEX_LOCK(&epi->epi_mutex);
 	D_ASSERT(epi->epi_req_num >= epi->epi_reply_num);
-	crt_set_timeout(rpc_priv);
```

Comment:
In CaRT we have a logic the timer is set when RPC is put into inflight queue. Here I just move it bellow to avoid set it twice. Because of it will be reset when RPC will be moved from waitq to inflightq.

Reply:
If so, I think it's unrelated to this patch.

@frostedcmos, do you think we should remove the crt_set_timeout call made when removing an RPC from the inflightq?

Reply:
Given that we want to avoid any risky changes at this point, and that this call is not expensive, i would prefer to revert it to how it was for purposes of this patch. Eventually it might be good idea to cleanup all such places where we set/reset timeouts

Reply:
ok. revert it.